CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Braintree Android SDK Release Notes
 
+## unreleased
+
+* PayPal
+  * Add `enablePayPalAppSwitch` property to `PayPalCheckoutRequest`
```

Comment:
Can we add an entry similar to what exists in `5.0.0`? That way we can more easily indicate this is in beta for the checkout flow as well.